### PR TITLE
UDS-1758: Make overview page optional

### PR DESCRIPTION
**Pull request by ggrimaudDTS**

### CODE REVIEW

File: @utahdts/utah-design-system-header/src/@types/jsDocTypes.d.js [hunk 1]
```diff
@@ -68,6 +68,7 @@
  *  @property {MenuItem[]} [actionMenu] - children menus
  *  @property {ChildrenMenuTypes} [childrenMenuType] - default is "flyout"
  *  @property {string} [className] - can be used for `selected` or any other purpose
+ *  @property {string} [hideOverview] - hide the "overview" page from the main menu (top of the list link)
```
Comment (AustinHaws-Utah) [inline @utahdts/utah-design-system-header/src/@types/jsDocTypes.d.js:71]: seems like this should be a boolean and not a string?

Comment (AustinHaws-Utah) [inline @utahdts/utah-design-system-header/src/@types/jsDocTypes.d.js:71]: boolean fields usually start with `is`. So maybe this property would be better named `isOverviewHidden`? `hideOverview` sounds like a function call?

File: utah-design-system-website/src/@types/jsDocTypes.d.js [hunk 2]
```diff
@@ -479,6 +479,7 @@
  *  @property {PageUrl[]} [parentLinks]
  *  @property {WebsiteMainMenuItem[]} [children]
  *  @property {ChildrenMenuTypes | null} [childrenMenuType]
+ *  @property {boolean} [hideOverview]
```
Comment (AustinHaws-Utah) [inline utah-design-system-website/src/@types/jsDocTypes.d.js:482]: same naming concern as above

File: @utahdts/utah-design-system-header/src/js/renderables/mainMenu/renderMainMenu.js [hunk 3]
```diff
@@ -120,9 +120,10 @@ export function renderMainMenu() {
         // if have both an action url and menu items, show a page link since the menu can't be both clicked to open
         // the sub menu AND clicked to got to the link
         if (
-          menuItem.actionFunction
+          (menuItem.actionFunction
```
Comment (AustinHaws-Utah) [inline @utahdts/utah-design-system-header/src/js/renderables/mainMenu/renderMainMenu.js:123]: ...not related to this line...
Mobile Main menu still shows the overview item, so need to update the mobile menu rendering to also skip the overview.

Comment (ggrimaudDTS) [reply @utahdts/utah-design-system-header/src/js/renderables/mainMenu/renderMainMenu.js:123]: Embarrassing...

**Pull request merged**